### ISSUE TITLE
pfpki: scheduled task to clean up revoked certificates

### ISSUE THREAD

**Issue by nqb** [inverse-inc/packetfence#6907]
**Is your feature request related to a problem? Please describe.**
Revoked certificates are kept forever. It could be interesting to have a scheduled task, configurable, which clean these certificates after a period of time (like we have for several items in DB).

**Describe the solution you'd like**
pfcron task.

**Comment by julsemaan** [comment]
just to make sure, this would be disabled by default like the rest of the cleanup tasks, correct?

**Comment by nqb** [comment]
Yes, I think. We should not try to automatically delete certificates (even if revoked).

**Comment by julsemaan** [comment]
> Yes, I think. We should not try to automatically delete certificates (even if revoked).

100% agree since it destroys the history of a device/user. Like with the node, person tables, we should let the admin pick his own retention policy and keep everything by default
